primitiv/device_ops/cuda/max.cu

```diff
@@ -49,10 +49,10 @@ __global__ void max_bw_dev(
   pgx += bid % skip + (bid / skip) * skip * n;
   argmax_val[tid] = n;
   for (std::uint32_t i = tid; i < n; i += BLOCK_SIZE) {
-    if (px[i * skip] == py[bid]) {
-      argmax_val[tid] = i;
-      break;
-    }
+    argmax_val[tid]
```

Comment:
The behavior of the false-case looks redundant; maybe it is enough to use if statement.